packages/cli/src/zed-integration/zedIntegration.ts

```diff
@@ -158,7 +158,7 @@ class GeminiAgent {
         );
         isAuthenticated = true;
       } catch (e) {
-        console.error(`Authentication failed: ${e}`);
+        debugLogger.error(`Authentication failed: ${e}`);
```

Comment:
this one needs to be user visible. also not sure about how this should be surfaced for the zed case. would follow up with them to understand if there is a way errors should be surfaced to them

Reply:
@mattKorwel should know or you can look at blame for this file to follow up with the github user who added this.